optuna_dashboard/static/components/DataGrid.tsx

```diff
@@ -313,7 +313,7 @@ function getComparator<T>(
     : (a, b) => -descendingComparator<T>(a, b, columns, orderBy)
 }
 
-function descendingComparator<T>(
+export function descendingComparator<T>(
```

Comment:
I think we should not export this function for unit tests.

Reply:
I see, we can implement a similar but simplified one in the test file.